### PR TITLE
[sc-4170] auth0 device code confirmation polling interval and timeout

### PR DESCRIPTION
**Pull request by steb-veksler**

### CODE REVIEW

File: pkg/auth/auth.go [hunk 1]
```diff
@@ -32,7 +30,9 @@ const (
 	GROUNDCOVER_AUTH_PATH                  = ".groundcover"
 	GROUNDCOVER_AUTH_FILE                  = "auth.json"
 	GROUNDCOVER_API_KEY_FILE               = "api_key.json"
-	AUTH0_MIN_INTERVAL                     = time.Second * 5
+	SPINNER_TYPE                           = 26 // ....
```
Comment (noamApps) [inline pkg/auth/auth.go:33]: why changing the spinner type?

**Review by noamApps — COMMENTED** [review]
if I understand correctly, this pr:
- changes the auth0 timeout to 1min
- auth0 polling uses spinner prompt - that means its now user facing ? if so, why is that? can you add screenshots?

File: pkg/auth/auth.go (hunk 1 above)
Comment (steb-veksler) [reply pkg/auth/auth.go:33]: I think the universal animation for waiting is (. .. ... )

**Comment by steb-veksler** [comment]
> 

spinner is not a prompt, just info and animation while waiting.
auth0 timeout is already 1 min.

**Comment by noamApps** [comment]
> > 
> 
> spinner is not a prompt, just info and animation while waiting. auth0 timeout is already 1 min.

but it looks like before the changes the timeout was 30 seconds and 1 minute

**Review by noamApps — APPROVED** [review]

**Pull request merged**